Library/Formula/monetdb.rb

```diff
@@ -11,17 +11,49 @@ class Monetdb < Formula
     sha1 "484c94edf77b0acb72aa0e0cb7a8017c149be95c" => :mountain_lion
   end
 
-  head "http://dev.monetdb.org/hg/MonetDB", :using => :hg
+  head do
+    url "http://dev.monetdb.org/hg/MonetDB", :using => :hg
 
-  option "with-java"
+    depends_on "libtool" => :build
+    depends_on "gettext" => :build
+    depends_on "automake" => :build
+    depends_on "autoconf" => :build
+  end
+
+  class RRequirement < Requirement
+    fatal true
+
+    satisfy { which('r') }
+
+    def message; <<-EOS.undent
+      R not found. The R integration module requires R.
+      Do one of the following:
+      - install R
```

Comment:
Yeh, that's fine. The problem is that the instructions as-is don't make it easy for people to install it. Perhaps we can say ```
- run `brew install homebrew/science/r` or `brew install Caskroom/cask/r`
```